Sources/Service/Environment/Environment+DotEnv.swift

```diff
@@ -0,0 +1,65 @@
+
+import Core
+#if os(Linux)
+    import Glibc
+#else
+    import Darwin
+#endif
+
+extension Environment {
+    /// Loads the ".env" and the ".env.environment" files if they exist
+    public func loadEnvironment() {
```

Comment:
Also, is there a reason to not just call this `load()`?

Reply:
I felt `load()` was just too broad.